### PR TITLE
Solving #4073 New PR 

### PR DESCRIPTION
**Pull request by j0rt3g4**
Substitutes PR #4244

### CODE REVIEW

**Comment by illfated** [comment]
Also: ref. issue ticket #4073
(PR titles do not create links)

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-authentication.md [hunk 1]
```diff
@@ -358,6 +358,9 @@ The following list shows the supported values:
 
 This policy is intended for use on Shared PCs to enable a quick first sign-in experience for a user. It works by automatically connecting new non-admin Azure Active Directory (Azure AD) accounts to the pre-configured candidate local accounts.
 
+> [!Important]
+> Pre-configured candidate local accounts: Means any **local account** (preconfigured or added) that you could have in the device.
```
Comment (nenonix) [inline windows/client-management/mdm/policy-csp-authentication.md]:
```suggestion
> Pre-configured candidate local accounts refers to any **local account** (pre-configured or added) in your device.
```

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #4073 

Thank you.

**Pull request merged**